tests/handlers/test_respond.py

```diff
@@ -0,0 +1,25 @@
+"""Test the respond handler."""
+
+import pytest
+
+from cactusbot.api import CactusAPI
+from cactusbot.handlers import ResponseHandler
+from cactusbot.packets import Packet, MessagePacket
+
+response_handler = ResponseHandler()
+
+
+@pytest.mark.asyncio
+async def test_user_update():
+    """Test the user update event."""
+
+    assert (await response_handler.on_username_update(Packet(username="TestUser"))
+           ) is None
```

Comment:
The fact that this returns `None` doesn't tell us too much in this case. We should also have `assert response_handler.username == "TestUser"` afterwards. :slightly_smiling_face: